### ISSUE TITLE
Color contrast check should be improved to take font size into account

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#21]
From Bill:

 Does the contrast issue take the "Large Font" option into account? I'm getting "Insufficient Color Contrast" on all buttons (https://a.cl.ly/OAuWyL0E) but WAVE says there are no errors because the text is large enough (https://a.cl.ly/RBu9DX5y).

**Comment by SteveJonesDev** [comment]
I just want to confirm these are the correct requirements to meet @amberhinds:

> WCAG 2.0 level AA requires a contrast ratio of at least 4.5:1 for normal text and 3:1 for large text.
> Large text is defined as 14 point (typically 18.66px) and bold or larger, or 18 point (typically 24px) or larger.

**Comment by amberhinds** [comment]
That looks correct to me.

**Comment by markaplet** [comment]
@SteveJonesDev I can confirm that yes you are correct.
